api_get.py

```diff
@@ -0,0 +1,66 @@
+from sanic import Sanic
```

Comment:
Let's assume that we have only 3 folders to work with:
1. Domain
2. Resources
3. Services

In your case I would like to see not "api_get.py" "api_post.py" but payment_resource.py. In this file there would be resource class with get and post inside. If you need more classes - you're welcome, but not many files.

Reply:
What files include the DOMAIN folder ?